airbyte-integrations/bases/base-normalization/unit_tests/resources/stripe_catalog_expected_nested_snowflake.json

```diff
@@ -0,0 +1,296 @@
+    "CHARGES_b34_EPS_ab1",
+    "CHARGES_b34_EPS_ab2",
+    "CHARGES_b34_EPS_ab3",
+    "CHARGES_bf0_FRAUD_DETAILS",
+    "CHARGES_bf0_FRAUD_DETAILS_ab1",
+    "CHARGES_bf0_FRAUD_DETAILS_ab2",
+    "CHARGES_bf0_FRAUD_DETAILS_ab3",
+    "CHARGES_c24_VISA_CHECKOUT",
+    "CHARGES_c24_VISA_CHECKOUT_ab1",
+    "CHARGES_c24_VISA_CHECKOUT_ab2",
+    "CHARGES_c24_VISA_CHECKOUT_ab3",
+    "CHARGES_c88_CARD_PRESENT",
+    "CHARGES_c88_CARD_PRESENT_ab1",
+    "CHARGES_c88_CARD_PRESENT_ab2",
+    "CHARGES_c88_CARD_PRESENT_ab3",
+    "CHARGES_cac_SHIPPING_ADDRESS",
+    "CHARGES_cac_SHIPPING_ADDRESS_ab1",
+    "CHARGES_cac_SHIPPING_ADDRESS_ab2",
+    "CHARGES_cac_SHIPPING_ADDRESS_ab3",
+    "CHARGES_da6_CARD",
+    "CHARGES_da6_CARD_ab1",
+    "CHARGES_da6_CARD_ab2",
+    "CHARGES_da6_CARD_ab3",
+    "CHARGES_ea6_SOFORT",
+    "CHARGES_ea6_SOFORT_ab1",
+    "CHARGES_ea6_SOFORT_ab2",
+    "CHARGES_ea6_SOFORT_ab3",
+    "CHARGES_ed0_BILLING_ADDRESS",
+    "CHARGES_ed0_BILLING_ADDRESS_ab1",
+    "CHARGES_ed0_BILLING_ADDRESS_ab2",
+    "CHARGES_ed0_BILLING_ADDRESS_ab3",
+    "CHARGES_f64_GIROPAY",
+    "CHARGES_f64_GIROPAY_ab1",
+    "CHARGES_f64_GIROPAY_ab2",
+    "CHARGES_f64_GIROPAY_ab3",
+    "COUPONS_ab1",
+    "COUPONS_ab2",
+    "COUPONS_ab3",
+    "CUSTOMERS_4d8_DISCOUNT",
+    "CUSTOMERS_4d8_DISCOUNT_ab1",
+    "CUSTOMERS_4d8_DISCOUNT_ab2",
+    "CUSTOMERS_4d8_DISCOUNT_ab3",
+    "CUSTOMERS_5a2_SHIPPING",
+    "CUSTOMERS_5a2_SHIPPING_ab1",
+    "CUSTOMERS_5a2_SHIPPING_ab2",
+    "CUSTOMERS_5a2_SHIPPING_ab3",
+    "CUSTOMERS_94e_CARDS",
+    "CUSTOMERS_94e_CARDS_ab1",
+    "CUSTOMERS_94e_CARDS_ab2",
+    "CUSTOMERS_94e_CARDS_ab3",
+    "CUSTOMERS_9ac_COUPON",
+    "CUSTOMERS_9ac_COUPON_ab1",
+    "CUSTOMERS_9ac_COUPON_ab2",
+    "CUSTOMERS_9ac_COUPON_ab3",
+    "CUSTOMERS_ab1",
+    "CUSTOMERS_ab2",
+    "CUSTOMERS_ab3",
+    "CUSTOMERS_c70_ADDRESS",
+    "CUSTOMERS_c70_ADDRESS_ab1",
+    "CUSTOMERS_c70_ADDRESS_ab2",
+    "CUSTOMERS_c70_ADDRESS_ab3",
+    "DISPUTES_682_EVIDENCE",
+    "DISPUTES_682_EVIDENCE_ab1",
+    "DISPUTES_682_EVIDENCE_ab2",
+    "DISPUTES_682_EVIDENCE_ab3",
+    "DISPUTES_ab1",
+    "DISPUTES_ab2",
+    "DISPUTES_ab3",
+    "DISPUTES_b7c_EVIDENCE_DETAILS",
+    "DISPUTES_b7c_EVIDENCE_DETAILS_ab1",
+    "DISPUTES_b7c_EVIDENCE_DETAILS_ab2",
+    "DISPUTES_b7c_EVIDENCE_DETAILS_ab3",
+    "DISPUTES_e74_BALANCE_TRANSACTIONS",
+    "DISPUTES_e74_BALANCE_TRANSACTIONS_ab1",
+    "DISPUTES_e74_BALANCE_TRANSACTIONS_ab2",
+    "DISPUTES_e74_BALANCE_TRANSACTIONS_ab3",
+    "EVENTS_ab1",
+    "EVENTS_ab2",
+    "EVENTS_ab3",
+    "INVOICES_270_DISCOUNT",
+    "INVOICES_270_DISCOUNT_ab1",
+    "INVOICES_270_DISCOUNT_ab2",
+    "INVOICES_270_DISCOUNT_ab3",
+    "INVOICES_71c_COUPON",
+    "INVOICES_71c_COUPON_ab1",
+    "INVOICES_71c_COUPON_ab2",
+    "INVOICES_71c_COUPON_ab3",
+    "INVOICES_ab1",
+    "INVOICES_ab2",
+    "INVOICES_ab3",
+    "INVOICE_ITEMS_ab1",
+    "INVOICE_ITEMS_ab2",
+    "INVOICE_ITEMS_ab3",
+    "INVOICE_IT_3ee_PERIOD",
```

Comment:
I feel it's a waste that snowflake (or bigquery) allows quite long identifiers (255 or 1024 characters) but we end up truncating them aggressively on those destinations and thus lose on readability, WDYT?